docker-compose.yml

```
@@ -110,5 +111,15 @@ services:
       - SSL_CERTFILE=/tls.cert
       - SSL_KEYFILE=/tls.key
     command: [ "Fulcrum", "/fulcrum.conf" ]
+  mempool-backend: 
+    image: mempool/backend
+    depends_on: [ bitcoind ]
+    environment:
+      MEMPOOL_BACKEND: "none"
+      CORE_RPC_HOST: bitcoind
+      CORE_RPC_PORT: "8332"
```

The RPC port is set to `18332` by default on regtest.
```suggestion
      CORE_RPC_PORT: "18332"
```